### ISSUE TITLE
github Releases notes not parsed

### ISSUE THREAD

**Issue by Fandekasp** [renovatebot/renovate#2638]
**What Renovate type are you using?**
Renovate CLI, Renovate GitHub App, or Renovate Pro?

**Describe the bug**
django-sendgrid-v5 has releases notes in its github releases page:
https://github.com/sklarsa/django-sendgrid-v5/releases

But it's not parsed and printed in renovate's PR
![](https://i.imgur.com/PKyi0yX.png)

Expecting github releases page to always be parsed and printed in renovate PRs. Isn't that already done for many packages ?

**Comment by rarkins** [comment]
That repo doesn't currently have any release notes for 0.7.0, so there's nothing to parse/print: https://github.com/sklarsa/django-sendgrid-v5/releases

The latest documented release is 0.6.88:

![image](https://user-images.githubusercontent.com/6311784/46789590-1b542a80-cd3d-11e8-81a1-c786a15e95f2.png)

There is also not even a tag so we cannot generate a compare URL between versions:
![image](https://user-images.githubusercontent.com/6311784/46789911-00ce8100-cd3e-11e8-931f-e9d39fe42c9f.png)

I've then taken a look at the JSON metadata for this release from pypi.org to see if it help us tie back to a commit in the source repo, but I don't think it does:

![image](https://user-images.githubusercontent.com/6311784/46790198-abdf3a80-cd3e-11e8-9d00-34cc6a0cbfe7.png)

In summary:
- There is no release notes for 0.7.0
- There is no tag for 0.7.0 for us to use for comparing source
- Pypi.org does not seem to include metadata about the source commits either

Given all that I think the only thing Renovate can/did do is link you to the source repo.
